docs/docusaurus.config.js

```diff
@@ -120,7 +120,16 @@ const config = {
       },
     }),
   // This is required to make the docs site searchable https://github.com/praveenn77/docusaurus-lunr-search
-  plugins: [require.resolve('docusaurus-lunr-search')]
+  plugins: [
+    require.resolve('docusaurus-lunr-search'),
+    [
+      '@docusaurus/plugin-google-gtag',
+      {
+        trackingID: 'XXX',
```

Comment:
this can be a GH secret and environment variable, also we should set this up for our staging environment and make sure this behaves as expected. We can discuss offline how to do this, but I have a feeling it involves setting up the docs firebase project.

Reply:
This is [relevant](https://stackoverflow.com/questions/49419250/is-sharing-google-analytics-tracking-id-safe); it seems these trackingIDs are all public and even if we make it a compiled GH secret, users will be able to view it on their browser by simply inspecting the source. 

The way we secure the gtag and prevent spam is we lock down what hostname is supported (ie `https://docs.urturn.app`). This makes setting up analytics in our staging/dev environment a little more complicated because hostnames are dynamic (we can figure this out in a followup).

We should at least verify this works in the PR using the auto-generated hostname https://board-games-dev-e8c04--pr147-yq-enable-ga-docs-1js85ci4.web.app/ created by our CI in this PR and connect with your own google-analytics tag temporarily.

This way we can verify what metrics/events we get and that it works. Let me know when you have the analytics setup and we can jump on a call to review what we get for free.